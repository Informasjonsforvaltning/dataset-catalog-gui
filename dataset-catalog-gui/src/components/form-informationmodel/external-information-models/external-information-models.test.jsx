import React from 'react';
import { shallow } from 'enzyme';
import ExternalInformationModels from './external-information-models.component';

test('should render ExternalInformationModels correctly', () => {
  expect(shallow(<ExternalInformationModels />)).toMatchSnapshot();
});
