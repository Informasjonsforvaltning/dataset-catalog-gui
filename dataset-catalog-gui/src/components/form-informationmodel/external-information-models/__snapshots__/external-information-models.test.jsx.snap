// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExternalInformationModels correctly 1`] = `
<div>
  <button
    className="fdk-btn-no-border"
    onClick={[Function]}
    type="button"
  >
    <CirclePlusInlineIcon />
  </button>
</div>
`;
